src/themes/default/elements/icon.overrides

```diff
@@ -353,16 +396,19 @@ i.icon.file.video.outline:before { content: "\f1c8"; }
 i.icon.file.code.outline:before { content: "\f1c9"; }
 
 /* Technologies */
-i.icon.barcode:before { content: "\f02a"; }
 i.icon.qrcode:before { content: "\f029"; }
+i.icon.barcode:before { content: "\f02a"; }
+i.icon.rss:before { content: "\f09e"; }
 i.icon.fork:before { content: "\f126"; }
 i.icon.html5:before { content: "\f13b"; }
 i.icon.css3:before { content: "\f13c"; }
-i.icon.rss:before { content: "\f09e"; }
 i.icon.rss.square:before { content: "\f143"; }
 i.icon.openid:before { content: "\f19b"; }
 i.icon.database:before { content: "\f1c0"; }
 i.icon.server:before { content: "\f233"; }
+i.icon.usb:before { content: "\f287"; }
+i.icon.bluetooth:before { content: "\f293"; }
+i.icon.bluetooth.alternative:before { content: "\f294"; }
```

Comment:
Three new Technology icons: `usb`, `bluetooth` and `bluetooth alternative` (that last one is the "B" without its background ellipse)